### PR TITLE
PFW-1297: Improve languages part 2

### PR DESCRIPTION
**Pull request by 3d-gussner**
- [x] No translation for  `Dump/Crash` messages
- [x] Delete duplicates
  - [x] `Choose filament` removed and reused `Select filament`
  - [x] Reuse `Extruder/Print fan:`
  - [x] Delete `Nozzle FAN`
 - [x] Change `Temp. cal.` to `PINDA cal.` 

Resource savings:
|Variant| before flash/mem | after flash/mem | diff flash/mem |
|---------|-------------------------|----------------------|---------------------|
|           | FW3.11.0-RC1     | FW3.11.0         |                         |
|MK3S |[250874/6110](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246394479#L306-L307)|  [250574/6105](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/247972733#L306-L307) | -300/-5
|MK3 |[251778/6138](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246394479#L561-L562)| [251268/6133](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/247973771#L590-L591) | -510/-5|
|MK25S |[223032/5973](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246394479#L816-L817)| [222132/5968](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/247973771#L872-L873)| -900/-5|

### CODE REVIEW

File: Firmware/messages.c [hunk 1]
```diff
@@ -52,7 +54,7 @@ const char MSG_HEATING_COMPLETE[] PROGMEM_I1 = ISTR("Heating done."); ////c=20
 const char MSG_HOMEYZ[] PROGMEM_I1 = ISTR("Calibrate Z"); ////c=18
 const char MSG_ITERATION[] PROGMEM_I1 = ISTR("Iteration"); ////c=12
 const char MSG_CHOOSE_EXTRUDER[] PROGMEM_I1 = ISTR("Choose extruder:"); ////c=20
```
Comment (DRracer) [inline Firmware/messages.c]: Would it make sense to change `choose extruder` to `select extruder` as well? Just to make it more consistent?

File: Firmware/ultralcd.cpp [hunk 2]
```diff
@@ -1438,14 +1438,9 @@ static void pgmtext_with_colon(const char *ipgmLabel, char *dst, uint8_t dstSize
...
 
-    // Display Nozzle fan RPM
     lcd_timeoutToStatus.stop(); //infinite timeout
     lcd_home();
-    static const size_t maxChars = 12;
-    char nozzle[maxChars], print[maxChars];
-    pgmtext_with_colon(_i("Nozzle FAN"), nozzle, maxChars);  ////MSG_NOZZLE_FAN c=10
-    pgmtext_with_colon(_i("Print FAN"), print, maxChars);  ////MSG_PRINT_FAN c=10
-	lcd_printf_P(_N("%s %4d RPM\n" "%s %4d RPM\n"), nozzle, 60*fan_speed[0], print, 60*fan_speed[1] ); 
+	lcd_printf_P(PSTR("%-16.16S%-4d\n" "%-16.16S%-4d\n"), _T(MSG_EXTRUDER_FAN_SPEED), 60*fan_speed[0], _T(MSG_PRINT_FAN_SPEED), 60*fan_speed[1] );
```
Comment (DRracer) [inline Firmware/ultralcd.cpp:1438]: :+1:

**Comment by DRracer** [comment]
nice savings :+1:

**Comment by 3d-gussner** [comment]
@Painkiller56 @pkg2000 Please do a final review of https://github.com/prusa3d/Prusa-Firmware/pull/3426/files#diff-cf46655e0192f28112da2985d28dc728949010730a0ac629dbb7049f9f53a357

lang/po/new/sv.po

**Comment by 3d-gussner** [comment]
@pkg2000 Please do a final review of https://github.com/prusa3d/Prusa-Firmware/pull/3426/files#diff-22e5dd5d54111e837f3b40db65f2a126b09cc4157dcfa1f34b6e5dde0bf4a607

lang/po/new/no.po

**Review by Painkiller56 — COMMENTED** [review]
I wonder if  you used the lastest sv.po ?

Anyway I have these changes, after that the translation is ok

#: ultralcd.cpp:8070
msgid "Loose pulley"
msgstr "Lös pulley"

# MSG_SELFTEST_SWAPPED c=16
#: ultralcd.cpp:8101
#: ultralcd.cpp:8083
msgid "Swapped"
msgstr "Utbytt"

# MSG_USERWAIT c=20
#: Marlin_main.cpp:4347
#: Marlin_main.cpp:4354
msgid "Wait for user..."
msgstr "Väntar på användare."


#: ultralcd.cpp:5641
#: ultralcd.cpp:5623
msgid "Checks"
msgstr "Kontroller"

**Review by DRracer — APPROVED** [review]
looks correct to me, approving

**Comment by 3d-gussner** [comment]
@Painkiller56 Thanks for the update, anything else?

**Pull request merged**